### ISSUE TITLE
Warning in Home Assistant: gehomesdk.erd.converters.specialized.erd_unit_type_converter

### ISSUE THREAD

**Issue by marshalltech81** [simbaja/ha_gehome#270]
In Home Assistant, I am receiving the following warning from the logs

<img width="490" alt="Screenshot 2024-06-27 at 1 40 36 PM" src="https://github.com/simbaja/ha_gehome/assets/68938002/d203bf2e-db3d-4a15-9d56-61fb2b96ea30">

**Comment by ziptbm** [comment]
I'm getting this too.  over 10k occurrences in just over 3 days.  

Logger: gehomesdk.erd.converters.specialized.erd_unit_type_converter
Source: runner.py:190
First occurred: July 26, 2024 at 07:57:44 (10232 occurrences)
Last logged: 21:31:16

Unknown Unit Type: 0000000B, using default

I reported this last year in [this thread](https://github.com/simbaja/ha_gehome/issues/204) and it was closed as not planed.
